p3/templatetags/p3.py

```diff
@@ -625,7 +625,7 @@ def match(e, exclude_tags=set(('partner0', 'partner1', 'sprint1', 'sprint2', 'sp
         return True
     events = { x['id']:x for x in filter(match, cdataaccess.events(conf=conference)) }
     talks = {}
-    for e in events.values():
+    for e in list(events.values()):
```

Comment:
Unnecessary `list`.

Reply:
For future reference: find . -type f -name \*.py -exec gawk -i inplace '{print gensub(/for (.+) in list\((.+)\):/, "for \\1 in \\2:", "g");}' {} \;